xain_fl/cli.py

```diff
@@ -12,34 +12,75 @@
 
 
 def type_num_rounds(value):
+    """[summary]
```

Comment:
Instead of contenting pylint by adding a dummy docstring I think we should turn the "missing docstring" lint into a warning. That way we can all work on fixing that progressively.

Reply:
i know that the dummy docstrings are not an optimal solution. the problem with warnings is, that the CI would still fail in its current configuration (that's why the `fixme` warning is disabled for now for example).
everytime people are working on parts of the code with dummy docstrings, they should replace them with proper docstrings, maybe we can add a jira task on the board and a todo in the code for that.

Reply:
Then can we have a greppable pattern to these dummy docstrings so that we can easily find them?

Reply:
agree. i created a ticket for that: https://xainag.atlassian.net/browse/XP-425. also, i added a line to the respective docstrings: `.. todo:: Advance docstrings (https://xainag.atlassian.net/browse/XP-425)` this will only be visible in the auto-generated documentation if the respective flag is set.